src/Faker/Provider/sq_AL/Internet.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Faker\Provider\sq_AL;
+
+class Internet extends \Faker\Provider\Internet
+{
+    protected static $freeEmailDomain = ['gmail.com', 'yahoo.com', 'hotmail.com'];
```

Comment:
No point in having these for a specific locale, either add specific Albanian domains or remove this.